newIDE/app/src/EventsSheet/ParameterFields/ObjectVariableField.js

```diff
@@ -88,12 +135,12 @@ export default React.forwardRef<ParameterFieldProps, ParameterFieldInterface>(
               : undefined
           }
         />
-        {editorOpen && variablesContainer && project && (
+        {editorOpen && variablesContainers.length === 1 && project && (
           <VariablesEditorDialog
             project={project}
             title={<Trans>Object Variables</Trans>}
             open={editorOpen}
-            variablesContainer={variablesContainer}
+            variablesContainer={variablesContainers[0]}
```

Comment:
A comment to say that the first item should always be the object's variables container?

Reply:
I don't think a comment is needed as there is a `variablesContainers.length === 1` condition around it.

Reply:
I had to think to know why you only take the first: it's the object's one. I can see myself from the future change the logic and not realize now it's another variables container that is not related to the object

Reply:
I added a comment.